ROADMAP MAR18-SEP18.md

```diff
@@ -13,46 +13,60 @@ Here are my current thoughts on a roadmap for our overall engineering organziati
 
-* Host docsify page on docs.operationcode.org - we need to have our docs in one easy to find and easy to navigate place, this is essential to making sure people can find the information they need quickly.  
+* ~~Host docsify page on docs.operationcode.org - we need to have our docs in one easy to find and easy to navigate place, this is essential to making sure people can find the information they need quickly.~~ _deprioritized, github will suffice for now -nell_
 
-* Landing Page ReDesign
+* *P1* Landing Page ReDesign _we need to communicate who we are and how you become a part of us, how to join. How do they know if we're legitimate? -davidm_
+ * Re-think and redo hero banner implementation
 
 * Introduce a drop-down item for the navbar.
 
-* Re-think and redo hero banner implementation
-
 * Make sure that all UI components have Jest snapshot tests and Storybook instances
+ * _in progress with kyle_
 
 * Document how to add a Jest test and a Storybook instance of a component
+ * _in progress with kyle_
 
 * Discuss moving shared components into UI Lib via NPM
+ * _check with kyle if this is still necessary -nell_
 
 ## Backend
 
-* Deploy interface for execs to query the database (possibly Active Admin or Forest)
+* ~~Deploy interface for execs to query the database (possibly Active Admin or Forest)~~ _done!_
  * As operations, I want the member list in the backend to contain both signup *and* Slack information, so that I can have a single view of our membership base.
 
-* Add ability to run Cron Jobs (need help of infrastructure)
+* ~~Add ability to run Cron Jobs (need help of infrastructure)~~
+ * _need more info, was requested by harry -ashley_
+ * _deprioritized until we have a backend lead - nell_
 
-* Set up new Code Climate account - this will help give us insight into the health of our codebase, and ensure we are using best practices and keeping it beginner friendly!
+* ~~Set up new Code Climate account - this will help give us insight into the health of our codebase, and ensure we are using best practices and keeping it beginner friendly!~~
+ * _deprioritized - nell_
 
-* Implement roles/permissions for end users
+* ~~Implement roles/permissions for end users~~ _done!_
 
 * Implement an authentication strategy/option to secure and expose the API (for non-signed in user consumers, i.e. Slack, apps, etc.)
+ * _need more info from long_common_name -nell_
```

Comment:
Need to follow up with @wimo7083 on this

Reply:
harry came up with a good idea for how to integrate pybot into BE. For now we can use that for this one case and treat pybot as a main focal point so I don't have to re-architect in ruby.